README.md

```diff
@@ -45,6 +45,7 @@
         - [Event Denoising](#denoising)
         - [Compression](#compression)
     - [Control](#control)
+    - [Obstacle Avoidance](#obstacle_avoidance)
```

Comment:
Should Obstacle Avoidance be a subsection within the Control section?

Reply:
I personally think that obstacle avoidance is a separated topic. Despite, it includes robot control, obstacle avoidance entails as well obstacle detection, trajectory estimation, mapping and planning (if necessary).